base/exports.jl

```diff
@@ -1149,6 +1149,7 @@ export
     ntoh,
     open,
     pipe,
+    Pipe,
```

Comment:
I think once processes are readable/writable, this can just be `pipe()` (although @vtjnash wants that to return a pair). I think maybe the right design is instead of have another method that creates the other pipe end.